databricks/koalas/indexes.py

```diff
@@ -675,6 +675,8 @@ def to_series(self, name: Union[str, Tuple[str, ...]] = None) -> Series:
         d    d
         Name: 0, dtype: object
         """
+        if not isinstance(name, abc.Hashable):
```

Comment:
Should we use pandas' `is_hashable` instead?

Reply:
Seems better. Thanks! 👍